alerting/provider/discord/discord_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/TwiN/gatus/v3/test"
 )
 
-func TestAlertProvider_IsValid(t *testing.T) {
+func TestAlertDefaultProvider_IsValid(t *testing.T) {
```

Comment:
Why is this renamed? We're still testing the method `IsValid` of the struct `AlertProvider` 🤔

Reply:
https://github.com/TwiN/gatus/pull/271/commits/9b764cfce29c1231da423fb1d5b531c241cf4bba done